doc/sphinx/proof-engine/tactics.rst

```diff
@@ -460,16 +460,14 @@ Examples:
 Occurrence sets and occurrence clauses
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-An occurrence clause is a modifier to some tactics that obeys the
+An occurrence clause (:n:`at @occurrences`) is a modifier to some tactics that obeys the
```

Comment:
```suggestion
An occurrence clause (:n:`in @occurrences`) is a modifier to some tactics that obeys the
```

Reply:
The grammar uses "at".  From fullGrammar:

```
| "rewrite" "*" orient uconstr "in" hyp "at" occurrences by_arg_tac
| "rewrite" "*" orient uconstr "at" occurrences "in" hyp by_arg_tac
| "rewrite" "*" orient uconstr "at" occurrences by_arg_tac
| "setoid_rewrite" orient glob_constr_with_bindings "at" occurrences
| "setoid_rewrite" orient glob_constr_with_bindings "at" occurrences "in" hyp
| "setoid_rewrite" orient glob_constr_with_bindings "in" hyp "at" occurrences
```

Reply:
These are not the only tactic you changed though. And most of the ones you changed use a more complex form. See answer below.